ml-testing-toolkit/Chart.yaml

```diff
@@ -1,8 +1,8 @@
 apiVersion: v1
 description:  ml-testing-toolkit Helm chart for Kubernetes
 name: ml-testing-toolkit
-version: 11.0.0
-appVersion: "10.4.1"
+version: 11.0.1
+appVersion: "backend 11.0.1 front-end 11.0.0"
```

Comment:
is this suppose to be 
`mojaloop/ml-testing-toolkit: v11.0.1; mojaloop/ml-testing-toolkit-ui: v11.0.0`

Reply:
Thanks Rajiv, I updated accordingly.